### PR TITLE
test: skip upstream after service is seleted

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
resolve #1302

### CODE REVIEW

File: web/cypress/integration/route/create-route-can-skip-upstream.spec.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/*
```
Comment (juzhiyuan) [inline web/cypress/integration/route/create-route-can-skip-upstream.spec.js:1]: bad title

Comment (juzhiyuan) [reply web/cypress/integration/route/create-route-can-skip-upstream.spec.js:1]: test what

**Review by juzhiyuan — APPROVED** [review]

File: web/cypress/integration/route/can-skip-upstream-when-select-service-id.spec.js [hunk 2]
```diff
@@ -18,19 +18,25 @@
 
 context('Can select service_id skip upstream in route', () => {
   const data = {
-    upstreamName: 'test_upstream',
-    serviceName: 'test_service',
-    routeName: 'test_route',
+    upstream_name: 'test_upstream',
```
Comment (LiteSun) [inline web/cypress/integration/route/can-skip-upstream-when-select-service-id.spec.js:21]: Why use UnderScoreCase？

Comment (juzhiyuan) [reply web/cypress/integration/route/can-skip-upstream-when-select-service-id.spec.js:21]: it's called snake_case and the API will return such a case, to reduce transformation, we allow to use that case.

**Review by LiteSun — APPROVED** [review]

**Pull request merged**